erigon-lib/state/merge.go

```diff
@@ -1082,4 +1082,12 @@ func hasCoverVisibleFile(visibleFiles []visibleFile, item *filesItem) bool {
 }
 
 func (ac *AggregatorRoTx) DbgDomain(idx kv.Domain) *DomainRoTx         { return ac.d[idx] }
-func (ac *AggregatorRoTx) DbgII(idx kv.InvertedIdx) *InvertedIndexRoTx { return ac.iis[idx] }
+func (ac *AggregatorRoTx) DbgII(idx kv.InvertedIdx) *InvertedIndexRoTx { return ac.searchII(idx) }
+func (ac *AggregatorRoTx) searchII(idx kv.InvertedIdx) *InvertedIndexRoTx {
+	for _, iit := range ac.iis {
```

Comment:
do you have any bench? maybe we not improving anything by this change? 
(FYI: `iit.ii.iiId` double pointer jump - can eliminate by copy `iiID` into `iit`).

Reply:
no bench. IMO cleaner to represent it as array rather than map. Array definitely better than map, even if not practically it's not much difference.

Reply:
ok. let's rename `iiId` to `name`. because `ID` word is reserved for auto_increment_id (and similar identifiers).